### ISSUE TITLE
Npm install doesn't include type definitions

### ISSUE THREAD

**Issue by jdforsythe** [protobufjs/protobuf.js#1201]
protobuf.js version: 5.0.3

After `npm install` there should be type definitions available.

In master, `package.json` appears to be configured for type definitions:

![Screen Shot 2019-04-15 at 3 01 58 PM](https://user-images.githubusercontent.com/286972/56158226-818d6480-5f8f-11e9-9b32-165c0de6cdfe.png)

However the process to release must be rewriting the file to remove the reference to `index.d.ts` and the type definition files are also excluded from the npm release completely.

This is perhaps the cause of https://github.com/grpc/grpc-node/issues/572

**Comment by JustinBeckwith** [comment]
Greetings!  This library is currently on version 6.8.8.  I just tested that version with TypeScript and all appears to be good.  Please try the latest version, and let us know if you're still running into trouble?

**Comment by nicolasnoble** [comment]
@JustinBeckwith: grpc still has a direct dependency on protobufjs 5, for API compatibility reasons.

**Comment by JustinBeckwith** [comment]
Apologies...  I had no idea!

**Comment by JustinBeckwith** [comment]
@nicolasnoble regardless though...  I wouldn't say this is a relevant bug on this repository, is it?  Seems to me it was fixed, we just can't upgrade to the version with the fix?

**Comment by nicolasnoble** [comment]
No, grpc can't upgrade to 6 without causing a breaking change to grpc itself. Now technically, the method used that gets people to use protobuf 5 with grpc is deprecated, but is also still around.

**Comment by dcodeIO** [comment]
Closing this issue because it has been inactive for more than two years and the project has changed substantially.

If this is still relevant, please open a fresh issue with current reproduction steps, version details, and any updated context.

Please also consider that maintaining protobuf.js takes time and effort. If the project is valuable to you or your organization, sponsorship is appreciated.

Thanks!
